sdk/servicebus/service-bus/src/serviceBusClient.ts

```diff
@@ -79,37 +79,59 @@ export class ServiceBusClient {
   }
 
   /**
-   * Creates a receiver for an Azure Service Bus queue.
+   * Creates a receiver for an Azure Service Bus queue in peekLock mode.
+   *
+   * In peekLock mode, the receiver has a lock on the message for the duration specified on the
+   * queue or subscription.
+   *
+   * If the message is not settled during this time it becomes available to other receivers.
+   *
+   * You can settle a message by calling complete(), abandon(), defer() or deadletter() methods on
+   * the message.
    *
    * @param queueName The name of the queue to receive from.
-   * @param receiveMode The receive mode to use (defaults to PeekLock)
+   * @param receiveMode The receive mode, defaulted to peekLock.
    */
   createReceiver(queueName: string, receiveMode: "peekLock"): Receiver<ReceivedMessageWithLock>;
   /**
-   * Creates a receiver for an Azure Service Bus queue.
+   * Creates a receiver for an Azure Service Bus queue in receiveAndDelete mode.
+   *
+   * In receiveAndDelete mode, messages are automatically removed from Service Bus as they are
```

Comment:
I don't see any additional value with `automatically` here. Also, may I recommend using the verb `delete` to match the mode?
```suggestion
   * In receiveAndDelete mode, messages are deleted from Service Bus as they are
```

Reply:
Fixed!

Reply:
Terminology absorbed.